tools/data_gen/src/main.rs

```diff
@@ -0,0 +1,197 @@
+#[macro_use]
+extern crate lazy_static;
```

Comment:
No need to use the old syntax, this can just be `use lazy_static::lazy_static`. Or alternatively the invocation on line 14 can just be fully qualified as `lazy_static::lazy_static! ...` and the import statement omitted. The latter might be preferable given that it is only needed in this one place.